[[Documentation]] maximize - title and zoombox

seaniepie opened the pull request:
- There doesn't seem to be a title property. However, the zoomBox property does exist.

Note: Zoom box (glossary) does not appear and is not linking to the page. I cannot see why this is as everything else seems ok.

seaniepie committed:
[[Documentation]] maximize - title and zoombox

- There doesn't seem to be a title property. However, the zoomBox property does exist.

Note: Zoom box (glossary) does not appear and is not linking to the page. I cannot see why this is as everything else seems ok.

livecodeali commented:
@seaniepie `title` is a synonym of `label`, so the correct fix there is to use the primary form (i.e. label). We should perhaps allow synonyms in the references, but for now it's probably easier to change to `label`.

seaniepie committed:
Changed title to label and added more tags

- Changed the 'title' tag to a value in the description and to label in the references.
- Added info about the label property
- More tags added for systemWindow, decorations and title bar

livecodeali commented:
@livecode-vulcan review ok e24d236

livecode-vulcan commented:
💙 review by @livecodeali ok e24d236b4d6f6be5aa169a7cc2f4d4476fa3cf4d

livecode-vulcan commented:
😎 test success e24d236b4d6f6be5aa169a7cc2f4d4476fa3cf4d
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/425)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/430)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/425)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/425)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/431)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/425)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/425)
